src/niamoto/core/plugins/exporters/html_page_exporter.py

```
@@ -1133,3 +1213,83 @@
                 exc_info=True,
             )
             return None
+
+    def _generate_traditional_index(
```

Pylint (3.3.7)

[refactor] Too many arguments (10/5)

(R0913)